Items/04_00_WeaponAttachments.txt

```diff
@@ -538,32 +585,35 @@ against the poison. Poison damage is taken at the end of the turn of the
 afflicted character.
 
 == RMM Match (Tournament) Grade Ammo  2x mag cost ==
+
     This high quality ammunition isn't cheap, but it increases the consistency 
 of your shots. 
 
-Ranged Miss Chance:
-*  -0.5
+    Ranged Miss Chance:  -0.5
 
 == High Velocity Ammunition  2.2x mag cost ==
-Longest Range Bracket:
-*  +10%m or +5m (whichever is higher)
 
-Base Damage:
-*  +10% (drop the remainder)
+    High Velocity Ammo uses slightly more dense metal and extra propellant to
+make bullets that jut go faster. This helps the ammo fly straighter and impart

```

Comment:
sp: "just"

Reply:
*embarrassed flopping on the ground and ugly-crying*